aws/resource_aws_eks_cluster.go

```diff
@@ -343,6 +391,40 @@ func expandEksVpcConfigUpdateRequest(l []interface{}) *eks.VpcConfigRequest {
 	}
 }
 
+func expandEksLoggingTypes(logTypes []interface{}) *eks.Logging {
+	enabled := true
+	disabled := false
+	enabledTypes := []*string{}
+	for _, logType := range logTypes {
+		typeVal := logType.(string)
+		enabledTypes = append(enabledTypes, &typeVal)
+	}
+
+	disabledTypes := []*string{}
+	for _, defaultType := range defaultLoggingTypes {
+		logType := defaultType
+		if _, ok := sliceContainsString(logTypes, logType); !ok {
+			disabledTypes = append(disabledTypes, &logType)
+		}
+	}
+
+	enabledLogs := &eks.LogSetup{
+		Enabled: &enabled,
```

Comment:
Nit: This can be simplified to

```suggestion
		Enabled: aws.Bool(true),
```